src/Deserializers/EntityDeserializer.php

```diff
@@ -12,10 +11,10 @@
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  * @author Thiemo Mättig
  */
-class EntityDeserializer implements Deserializer {
+class EntityDeserializer implements DispatchableDeserializer {
```

Comment:
As far as I can see this could be replaced by a `DispatchingDeserializer` so that this class gets obsolete.